pkgs/top-level/python-packages.nix

```diff
@@ -11890,6 +11890,20 @@ in modules // {
     };
   });
 
+  libgpuarray-cuda = callPackage ../development/python-modules/libgpuarray/cuda/default.nix rec {
+    numpy = self.numpy_1_10;
+    scipy = self.buildScipyPackage rec {
+      version = "0.17.1";
+      src = pkgs.fetchurl {
```

Comment:
I should have been clearer there. What I meant is that the source has already been specified, so doing a `scipy.override{numpy=self.numpy_1_10};` would be sufficient here.